darwin/DarwinProcessList.c

```diff
@@ -192,7 +194,7 @@ void ProcessList_goThroughEntries(ProcessList* super, bool pauseProcessUpdate) {
       }
    }
 
-   const double time_interval = ticksToNanoseconds(dpl->global_diff) / (double) dpl->super.activeCPUs;
+   const double time_interval_ns = schedulerTicksToNanoseconds(dpl->global_diff) / (double) dpl->super.activeCPUs;
```

Comment:
The values that come back from `host_processor_info` are *NOT* in Mach timebase tick units.

They're in "scheduler ticks" (as I call them), which are still the same 100 ticks/second on M1 as they used to be on Intel Macs.